napari/components/_layer_slicer.py

```diff
@@ -153,7 +153,7 @@ def slice_layers_async(
         sync_layers = []
         for layer in layers:
             if isinstance(layer, _AsyncSliceable) and not self._force_sync:
-                logger.debug(f'Async slicing {layer.name}')
+                logger.debug('Async slicing: %s', layer)
```

Comment:
I tend to use printf style logs because I'm worried that f-strings are being eagerly evaluated (so this line is no longer a no-op). Happy to change back if there's documentation that they are lazily evaluated though.